lib/src/message_options.dart

```diff
@@ -35,18 +35,19 @@ class MessageOptions {
   /// Create message with given media stream.
   ///
   /// If you specify [MessageOptions.withMedia] then you will not be able to specify [MessageOptions.withBody] because they are mutually exclusive message types. Created message type will be [MessageType.MEDIA].
-  void withMedia(File input, String mimeType) {
+  void withMedia(List<File> input, List<String> mimeType,List<String> filename) {
```

Comment:
Hi, thanks for the feature @manandharsudhir,

This line will cause a breaking change since the parameters changed to lists.  This is evident when compiling the project example. Is there a way to keep the method paramaters the same and chain `addMedia` calls?  See how this [builder](https://www.twilio.com/docs/conversations/sending-messages-and-media#using-message-builder) from the Twilio docs adds multiple files.  

In addition, the `filename` used to be optional.  This might be another bug since it's not used on Kotlin and on Swift it's hardcoded to `image.jpg`.  Is it possible to keep it optional and apply defaults if it was not provided? 

And finally, this feature has to be applied on the Android plugin as well so that both platforms have the same implementation.